extensions/ql-vscode/src/view/results/locations/ClickableLocation.tsx

```diff
@@ -8,7 +8,7 @@ interface Props {
   label: string;
   databaseUri: string;
   title?: string;
-  jumpToLocationCallback?: () => void;
+  handleClick?: () => void;
```

Comment:
Could we call this `onClick` instead? Sorry if I was unclear about which part to rename, but the general convention I like to use is that the prop is called `on...` and if we're defining something using `useCallback`, it's called `handle...`.

Reply:
Ah, sorry I misunderstood. Thanks for keeping us in line with the general conventions.